website/content/docs/developers/packing/ffdpacking.md

```diff
@@ -0,0 +1,61 @@
+---
+title: The First Fit Decreasing Packing Algorithm
+---
+
+### Overview
+
+This guide provides basic steps at using and tuning the First Fit Decreasing packing algorithm in order to  utilize the resources efficiently. This packing algorithm aims at utilizing as few containers as possible, thus limiting the overall resources used. The algorithm is based on the First Fit Decreasing heuristic for the binpacking problem. To enable the First Fit Decreasing algorithm, update the `packing.yaml` file as follows:
+
+```yaml
+# packing algorithm for packing instances into containers
+heron.class.packing.algorithm:    com.twitter.heron.packing.binpacking.FirstFitDecreasingPacking
+```
+
+The algorithm accepts as input the values of the following parameters:
+
+1. Component RAM 
+2. Hint for the maximum container RAM (`com.twitter.heron.api.Config.TOPOLOGY_CONTAINER_MAX_RAM_HINT`)
+3. Hint for the maximum Container CPU (`com.twitter.heron.api.Config.TOPOLOGY_CONTAINER_MAX_CPU_HINT`)
+4. Hint for the maximum Container Disk (`com.twitter.heron.api.Config.TOPOLOGY_CONTAINER_MAX_DISK_HINT`)
```

Comment:
In #3 and #4 Container is capitalized but other occurrences in the doc are now. We should be consistent.